sdk/physical/cache.go

```diff
@@ -146,6 +148,7 @@ func (c *Cache) Put(ctx context.Context, entry *Entry) error {
 	err := c.backend.Put(ctx, entry)
 	if err == nil {
 		c.lru.Add(entry.Key, entry)
+		metrics.IncrCounter([]string{"cache.write"}, 1)
```

Comment:
These should be in the form []string{"cache","write"}.

I believe this is because not all metrics backends concatenate with a period.